### PR TITLE
Deploy MVP demo of Club-check-in

### PR DESCRIPTION
**Pull request by wangwenyue**
Replace BrowserRouter with HashRouter.

Modify some files formatt.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,14 +22,5 @@
   },
   "eslintConfig": {
     "extends": "react-app"
-  },
-  "browserslist": [
```
Comment (Keith-CY) [inline package.json:27]: why removed these

File: src/components/Header/index.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ import {
 class Header extends React.Component {
 
     render() {
-        const {titleName} = this.props.titleName
+        const {titleName} = this.props
```
Comment (Keith-CY) [inline src/components/Header/index.js]: spaces around `titleName`

File: src/containers/Detail/index.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ import './detail.css'
 @inject('detailStore') @observer
 class Detail extends React.Component {
 
+
     render() {
 
         const {
```
Comment (Keith-CY) [inline src/containers/Detail/index.js]: This component could be moved outside

File: src/containers/Personal/ClubListItem/index.js [hunk 4]
```diff
@@ -11,34 +12,36 @@ class ClubListItem extends React.Component {
...
-                        <div className='club-list-name'>
-                            <span>{data['clubName']}</span>
+                <Link to={'/detail'}>
+                    <Flex onClick={handleClubDetail} key={index} className='club-list-item'>
+                        <div className='club-list-item-left'>
+                            <div className='club-list-name'>
+                                <span>{data['clubName']}</span>
+                            </div>
+                            <ul>
+                                <li>{<img src={data['avatar'][0]} alt=""/>}</li>
```
Comment (Keith-CY) [inline src/containers/Personal/ClubListItem/index.js]: use array.map to render thses elements

**Review by Keith-CY — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (wangwenyue) [reply package.json:27]: dependency problem, has to be removed.
More description can refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/1)

**Comment by wangwenyue** [comment]
@Keith-CY Resolve all request.

File: config-overrides.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
 const rewireMobX = require("react-app-rewire-mobx")
-const { injectBabelPlugin } = require('react-app-rewired')
+const {injectBabelPlugin} = require('react-app-rewired')
```
Comment (Keith-CY) [inline config-overrides.js]: space around `injectBabelPlugin`

**Review by Keith-CY — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (wangwenyue) [reply package.json]: This problem has been solved, more detail refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/2)

Comment (wangwenyue) [reply package.json:27]: > dependency problem, has to be removed.
> More description can refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/1)

This problem has been solved, more detail refer to this [issue](https://github.com/wangwenyue/wangwenyue.github.io/issues/2)

**Pull request merged**